src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import tagComponent, {
+  TagProps,
+} from "../../../__internal__/utils/helpers/tags";
+import StyledCrumb from "./crumb.style";
+
+export interface LinkProps extends TagProps {
```

Comment:
This is incorrect. The way you had it previously was closer to what we need, something like this would work as Link already supports the interface

```
export interface CrumbProps extends Omit<LinkProps, "toolTipMessage", "tooltipPosition">, TagProps {
 /** This sets the Crumb to current, does not render Link */
  isCurrent?: boolean;
}
```

Reply:
Adding @edleeks87's suggestion should also fix that lint error for you 👍🏼

Reply:
Sorry @edleeks87 - I misunderstood initially your request.